src/models/applicationState.ts

```diff
@@ -371,7 +371,7 @@ export enum FieldType {
     Date = "date",
     Time = "time",
     Integer = "integer",
-    Checkbox = "checkbox",
+    SelectionMark = "selection mark",
```

Comment:
selectionMark

Reply:
We also need to provide backward compatibility, in projectService.ts:getTagsFromPreExistingFieldFile, after we read fieldInfo from fields.json, we can convert all checkbox to selectionMark.
And make sure we test the existing fields.json still works